SETUP.md

```
@@ -162,12 +165,14 @@ Before using the system, index the PDF documents into Qdrant:
 python scripts/ingest.py ./archives/
 
 # Or index a specific file
-python scripts/ingest.py ./archives/agricultural_document.pdf
+python scripts/ingest.py ./archives/smart_boletim.pdf
 ```
 
-> **Alternative**: Use the semantic chunker directly:
+> **Alternative**: call the semantic chunker as a module, so the repository root stays on
+> `sys.path`. Running it as a file path (`python database/semantic_chunker.py`) fails with
+> `ModuleNotFoundError: No module named 'retrieval'`.
 > ```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)